zerver/openapi/zulip.yaml

```diff
@@ -14668,6 +14674,17 @@ paths:
             type: boolean
           example: false
           required: false
+        - name: is_default_stream
+          in: query
+          description: |
+            Add or remove the stream as a [default stream][default-stream]
+            for new users joining the organization.
+
+            [default-stream]: /help/set-default-streams-for-new-users
```

Comment:
You need to add `**Changes**` entry here as mentioned in last point [here](https://zulip.readthedocs.io/en/latest/documentation/api.html#step-by-step-guide).

Reply:
This should be done for all the API changes.